spec/features/user_peeps_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature "So that I can let people know what I am doing" do
+  scenario "I want to post a peep" do
+    visit '/'
+    if !page.has_content?('We have no peeps yet...')
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)